.eslintrc

```diff
@@ -10,6 +10,22 @@
   },
   "rules": {
     "@wordpress/dependency-group": "error",
-    "@wordpress/react-no-unsafe-timeout": "error"
+    "@wordpress/react-no-unsafe-timeout": "error",
+    "comma-dangle": [
+      "error",
+      {
+        "arrays": "always-multiline",
+        "objects": "always-multiline",
+        "imports": "always-multiline",
+        "exports": "always-multiline",
+        "functions": "never"
```

Comment:
A trailing comma after the second argument looks strange to me: `doThis( 'arg1', 'arg2', )`

...and it doesn't have great [browser support](https://caniuse.com/#feat=mdn-javascript_grammar_trailing_commas_trailing_commas_in_functions). This `.eslintrc` is also used for a non-compiled file: `admin.block-post.js`. 

Ideally, soon that file will be ported to the React logic in the refactoring.